enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -1007,31 +1008,61 @@ func (s *ExecutionServer) PublishOperation(stream repb.Execution_PublishOperatio
 			return err
 		}
 
+		if op.GetName() == "" {
```

Comment:
this change feels a little risky, is it needed?

Reply:
I could revert this change, but as far as I can tell, there's no way for PublishOperation to do anything useful without the name being an execution ID.